### ISSUE TITLE
Pull to refresh

### ISSUE THREAD

**Issue by ramona-cristea** [mikepenz/MaterialDrawer#2152]
Hello, 

Could you please tell me if this library supports Pull to Refresh inside the MaterialDrawer? 
Is it possible to achieve such a behavior? Do I have to implement a custom layout for the Drawer and add the DrawerLayout widget inside a SwipeRefreshLayout widget?

**Comment by mikepenz** [comment]
@ramona-cristea this behaviour is not included, but you theoretically can achieve it. 
The `MaterialDrawer` internally uses a `RecyclerView` to host it'S item. 

https://github.com/mikepenz/MaterialDrawer/blob/6a53d7631fa5f1ca8c1cbcd92cef36d806826149/library/src/main/res/layout/material_drawer_recycler_view.xml

You can copy paste this *.xml to your project. wrap the `RecyclerView` in a `SwipeRefreshLayout` (make sure the id of the RV stays the same) give your `SwipeRefreshLayout` and `id` 
and then access it after you built the drawer. 

basically find your `SwipeRefreshLayout` view by id from this parent:
https://github.com/mikepenz/MaterialDrawer/blob/6a53d7631fa5f1ca8c1cbcd92cef36d806826149/library/src/main/java/com/mikepenz/materialdrawer/Drawer.java#L194

**Comment by ramona-cristea** [comment]
I will try that. Thanks a lot!
